config/locales/en.yml

```diff
@@ -1332,7 +1333,11 @@ en:
         Do you really want to close the order %{order}?
         There is no going back.
       create_invoice: Add invoice
-      description1: "%{state} order from %{supplier} created by %{who}, open from %{starts} until %{ends}."
+      description1_order: "%{state} order from %{supplier} created by %{who}, "
+      description1_period:
+        starts: "open from %{starts}"
+        starts_ends: "open from %{starts} until %{ends}"
+        pickup: " and can be picked up on %{pickup}"
```

Comment:
let's put the leading space in the code - can be hard for translators to see at all